diff --git a/node_modules/vue-pdf/src/CMapReaderFactory.js b/node_modules/vue-pdf/src/CMapReaderFactory.js
index 85be2aa..dadd8f7 100644
--- a/node_modules/vue-pdf/src/CMapReaderFactory.js
+++ b/node_modules/vue-pdf/src/CMapReaderFactory.js
@@ -8,7 +8,7 @@ export default function() {
 
 		return import('./buffer-loader!pdfjs-dist/cmaps/'+query.name+'.bcmap' /* webpackChunkName: "noprefetch-[request]" */)
 		.then(function(bcmap) {
-
+      delete require.cache[require.resolve('./buffer-loader!pdfjs-dist/cmaps/'+query.name+'.bcmap')]
 			return {
 				cMapData: bcmap.default,
 				compressionType: CMapCompressionType.BINARY,
diff --git a/node_modules/vue-pdf/src/buffer-loader.js b/node_modules/vue-pdf/src/buffer-loader.js
index 76cd096..f7764a8 100644
--- a/node_modules/vue-pdf/src/buffer-loader.js
+++ b/node_modules/vue-pdf/src/buffer-loader.js
@@ -1,5 +1,5 @@
 var loaderUtils = require('loader-utils');
-
+const { Buffer } = require("buffer")
 module.exports = function(content) {
 
 	var options = loaderUtils.getOptions(this);
@@ -10,7 +10,8 @@ module.exports = function(content) {
 	else
 		data = Buffer.from(content);
 
-	return 'export default Buffer.from("'+data.toString('base64')+'", "base64")';
+	// return 'export default Buffer.from("'+data.toString('base64')+'", "base64")';
+	return 'const { Buffer } = require("buffer"); export default Buffer.from("'+data.toString('base64')+'", "base64")';
 }
 
 module.exports.raw = true;
